pkg/event-manager/controller.go

```diff
@@ -15,8 +15,8 @@ import (
 
 // Event manager constants
 const (
-	DefaultResyncPeriod = 60 * time.Second
-	DefaultWorkerNumber = 100
+	defaultResyncPeriod = 60 * time.Second
```

Comment:
60 is too long...

Reply:
this is only name change, value is the same ;) I can update that of course.